ansible/load_env_vars.yml

```diff
@@ -8,9 +18,9 @@
 # Bootstrapping the acl stanza configures the Nomad agent to enable ACLs and tunes various ACL parameters.
 # Either true or false
 # https://www.nomadproject.io/docs/configuration/acl
-- name: Set nomad acl to "false" if it is not set by user
-  set_fact: nomad_agent_acl_enable="false"
-  when: nomad_agent_acl_enable is undefined
+- name: Set nomad acl equal to nomad_acl from /vagrant/env_default.yml if not set some other place beforehand
+  set_fact: nomad_acl="{{ nomad_acl }}"
+  when: nomad_acl is undefined
```

Comment:
```suggestion
```
I'm not loving this part. With an env_default file it's not intuitive that the variables are also set from within ansible. Should we just remove this?